docs/API.rst

```diff
@@ -34,6 +35,8 @@ These are simple plots that inherit from the BasePlot class.
    VLinePlot
    ScatterPlot
 
+.. _core-mass-spec-plots:
+
```

Comment:
This section is not rendering properly now. Perhaps because ..currentmodule:: line was deleted? 

This is what it should look like: 

![image](https://github.com/user-attachments/assets/358ee03a-df8a-4276-95c6-41477b7dc228)

This is what yours looks like:

![image](https://github.com/user-attachments/assets/89b78ae4-c9d6-4ddf-8946-92544cd208c4)